pending/mpc-hc.xml

```diff
@@ -1,54 +1,139 @@
-    Web site    :  https://mpc-hc.org/
-
-
-Testing environment
-
-Build information:
-    Version:            1.7.1 (9ae7b7c)
-    Compiler:           MSVC 2012 Update 4 (SSE)
-    Build date:         Nov 17 2013 09:24:49
-
-LAV Filters:
-    LAV Splitter:       0.59.1.26
-    LAV Video:          0.59.1.26
-    LAV Audio:          0.59.1.26
-    FFmpeg compiler:    MinGW-w64 GCC 4.8.2
-
-Operating system:
-    Name:               Windows NT 6.1 (build 7601, Service Pack 1)
-    Version:            6.1 (64-bit)
-
--->
-<cleaner id="mpc-hc" os="windows">
-  <label>MPlayer-HC</label>
-  <description>Media player</description>
-  <option id="history">
-    <label>Most recently used</label>
-    <description>Delete the most recently used list</description>
-    <action command="winreg" path="HKCU\Software\MPC-HC\MPC-HC\Recent File List"/>
-    <action command="delete" search="file" path="$APPDATA\MPC-HC\default.mpcpl"/>
-  </option>
-</cleaner>
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2019 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+    @app MPC-HC/MPC-BE (Media Player Classic)
+    @url https://mpc-hc.org/
+    @os Windows
+    @cleanerversion v0.4.5
+    @cleanerdate 2019-03-18
+    @cleanerby 8ohmh (2016-03-23) & Tobias B. Besemer (2018-12-20 - 2019-03-18)
+    @tested ok v1.7.1 (9ae7b7c), Windows NT 6.1 (build 7601, Service Pack 1) (64-bit)
+    @testeddate ??? (... with an old version of the cleaner!)
+    @testedby 8ohmh
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-11 (... with an old version of the cleaner!)
+    @testedby https://github.com/Tobias-B-Besemer
+    @note That we don't need so much translation, we combine some options in "History"! Tobias.
+
+-->
+<cleaner id="mpc-hc" os="windows">
+  <label>MPC-HC/MPC-BE (Media Player Classic)</label>
+  <description>Media player</description>
+  <!-- That we don't need so much translation, we combine some options in "History"! Tobias. -->
+  <option id="history">
+    <label>History</label>
+    <description>Delete the usage history</description>
+  <!-- That we don't need so much translation, we combine some options in "History"! Tobias.
+  <option id="mru">
+    <label>Most recently used (MRU)</label>
+    <description>Delete the most recently used (MRU) list</description>
+  -->
```

Comment:
In case you want to use them, these two strings are already translated

> Most recently used

> Delete the most recently used list

Reply:
As the cleaner deletes in this option ""DVD Position [?]", too, I decided to use `History` & `Delete the usage history`. But I haven't pushed my last version so far... :-/